gallery_app/lib/widgets/row_column_expanded_widgets/row_column_menu.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gallery_app/modules/row_column_expanded/cubit/row_column_expanded_cubit.dart';
+import 'package:gallery_app/modules/row_column_expanded/model/row_column_expanded_state.dart';
+
+class RowColumnMenu extends StatelessWidget {
+  const RowColumnMenu({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final RowColumnCubit rowColumnCubit =
+        BlocProvider.of<RowColumnCubit>(context);
+    return BlocBuilder<RowColumnCubit, RowColumnState>(
+      builder: (context, state) {
+        return Container(
+          padding: const EdgeInsets.all(10),
+          color: Colors.white,
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  Row(
+                    children: [
+                      Radio(
+                        activeColor: Colors.blue,
+                        value: "Row",
+                        groupValue: state.type,
+                        onChanged: (value) {
+                          rowColumnCubit.setType(value.toString());
+                        },
+                      ),
+                      const Text("Row")
+                    ],
+                  ),
+                  Row(
+                    children: [
+                      Radio(
+                        activeColor: Colors.blue,
+                        value: "Column",
+                        groupValue: state.type,
+                        onChanged: (value) {
+                          rowColumnCubit.setType(value.toString());
+                        },
+                      ),
+                      const Text("Column")
+                    ],
+                  ),
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  const Text(
+                    "mainAxisSize",
+                    style: TextStyle(fontSize: 18),
+                  ),
+                  DropdownButton(
+                    value: state.mainAxisSize,
+                    items: [MainAxisSize.max, MainAxisSize.min].map((value) {
```

Comment:
Use MainAxisSize.values => It returns a list of values contained in an enum type